docs/administrators.rst

```diff
@@ -264,27 +264,49 @@ Locations
    :width: 80%
    :alt: Storage Service locations screen.
 
-A storage Location is contained in a Space, and knows its purpose in the
-Archivematica system. A Location is also where Packages are stored. Each
-Location is associated with a pipeline and can only be accessed by that
+A storage Location is contained in a Space, and knows its purpose in the Archivematica system.
+Each Location is associated with a pipeline and can only be accessed by that pipeline.
+Currently, a Location can have one of eight purposes: Transfer Source, Transfer Backlog, AIP Storage, DIP Storage, Currently Processing, Storage Service Internal Processing, AIP Recovery or FEDORA Deposit.
+
+Transfer source locations display in Archivematica's Transfer tab, and any folder in a transfer source can be selected to become a Transfer.
+The default value is '/home' in a Local Filesystem.
+This is required to start transfers.
+
+Transfer backlog stores transfers until such a time that the archivist contiues
```

Comment:
Amusingly, that was a line that was there originally that I just moved around.  Fixed. :)